model/patch/db.go

```diff
@@ -331,7 +330,6 @@ func FindFailedCommitQueuePatchesInTimeRange(projectID string, startTime, endTim
 				{StartTimeKey: bson.M{"$gte": startTime}},
 			}},
 			{"$and": []bson.M{
-				{StartTimeKey: time.Time{}},
```

Comment:
This might be possible if you call `MarkEnd` before the task actually starts. Container tasks currently call `MarkEnd` to system fail the task if the container can't run (meaning the task can't start) so they have a finish time but no start time.